src/HexagonOptimize.cpp

```diff
@@ -1443,6 +1461,10 @@ class EliminateInterleaves : public IRMutator2 {
     };
     Scope<BufferState> buffers;
 
+    // True for buffers that have loads and stores that are aligned
```

Comment:
Maybe more correct to say

    // False for buffers that have any loads or stores that are unaligned

? (existing is a bit ambiguous: all loads or some loads?)